CMakeLists.txt

```diff
@@ -149,15 +149,19 @@ if( MIOPEN_BACKEND STREQUAL "OpenCL")
     find_package( OpenCL REQUIRED )
     find_program(MIOPEN_HIP_COMPILER hcc
             PATH_SUFFIXES bin
-            PATHS /opt/rocm
+            PATHS 
+	        /opt/rocm
+		${CMAKE_INSTALL_PREFIX}
     )
     set(MIOPEN_USE_MIOPENGEMM ON CACHE BOOL "")
     if(MIOPEN_HIP_COMPILER)
         message("hip compiler: ${MIOPEN_HIP_COMPILER}")
     else()
         find_program(MIOPEN_HIP_COMPILER clang++
             PATH_SUFFIXES bin
-	        PATHS /opt/rocm/llvm
+	        PATHS 
+		    /opt/rocm/llvm
+		    ${CMAKE_INSTALL_PREFIX}/llvm
```

Comment:
So the `/opt/rocm/llvm` can be removed as its already added to the `CMAKE_PREFIX_PATH` on line 110. We should probably add `${CMAKE_INSTALL_PREFIX}/llvm` to the CMAKE_PREFIX_PATH as well and remove adding these extra paths everywhere.